libs/application/templates/parental-leave/src/parentalLeaveUtils.ts

```diff
@@ -310,3 +311,141 @@ export const calculatePeriodPercentage = (
 
   return Math.min(100, Math.round((months / difference) * 100))
 }
+
+type OtherParent = typeof NO | typeof MANUAL | undefined
+
+export function getApplicationAnswers(answers: Application['answers']) {
```

Comment:
I like this move. I saw in a few file we can get the type directly from `SchemaFormValues['otherParent']`, I guess it would be nice to use that instead of our types cast